src/lib/models/member.ts

```diff
@@ -0,0 +1,22 @@
+import * as v from 'valibot';
+
+export const committees = [
+    'Executive',
+    'Innovation',
+    'Service',
+    'External Relations',
+    'Membership & Internals',
+    'Branding & Creatives',
+    'Engineering',
+];
```

Comment:
As a matter of convention, I prefer naming (true) constants in `SCREAMING_CASE` just to signify that the values are not your typical variables in `camelCase`. I also suggest adding an `as const` cast to the data so that TypeScript infers the array as [`readonly`](https://www.typescriptlang.org/docs/handbook/utility-types.html#readonlytype).

```suggestion
export const committees = [
    'Executive',
    'Innovation',
    'Service',
    'External Relations',
    'Membership & Internals',
    'Branding & Creatives',
    'Engineering',
] as const;
```